sharp/CmdLaunch.cs

```diff
@@ -64,6 +64,17 @@ public unsafe class CmdLaunch : Cmd<string, string, bool, string> {
 
             if (!string.IsNullOrEmpty(args))
                 game.StartInfo.Arguments = args;
+        
+            // Flatpak detection
+            // or string.Equals(Environment.GetEnvironmentVariable("container"), "flatpak");
+            if (File.Exists("/.flatpak-info")) {
+            	if (!string.IsNullOrEmpty(args))
+                    game.StartInfo.Arguments = string.Join(" ", game.StartInfo.FileName, args);
+                else
+                    game.StartInfo.Arguments = game.StartInfo.FileName;
```

Comment:
does that work if the game has spaces somewhere in its path?

Reply:
I just tested this special case and it's not working when the path contains space somewhere, a fix/workaround could be `string.Join(" ", "\"" + game.StartInfo.FileName + "\"", args)`